<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>moreWindow</class>
 <widget class="QMainWindow" name="moreWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Подробности</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>791</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="worknameLabel">
       <property name="text">
        <string>Название работы:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fieldLabel">
       <property name="text">
        <string>Область знания:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="cafedraLabel">
       <property name="text">
        <string>Кафедра:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="consistencyLabel">
       <property name="text">
        <string>Состав кафедры:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="consistencyTableWidget">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="costLabel">
       <property name="text">
        <string>Затраты:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="backButton">
         <property name="text">
          <string>Назад</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editButton">
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
